app/views/users/_edit_form.html.erb

```diff
@@ -1,5 +1,6 @@
 <div style="padding:10px;">
 <%= form_for @user, :url => {:controller => 'users', :action => 'update'}, :html => {:class => "row"} do |f| %>
+  <%= hidden_field_tag "user[ui_update]", "0" %>
```

Comment:
Hmm, couldn't this be modified locally to be something other than 0, say, with javascript? Just curious, what is this bypass for? Thanks!

Reply:
I added the ui_update flag to make sure the update action was called from edit profile page. One of the tests of update action were failing so I thought that controller was used in more than one place. I could change it's value to something like ..false or true.. I did not get what you meant by changing with javascript could you explain this to me ? Thanks